test/test.js

```diff
@@ -52,10 +70,10 @@ async function assertFilesEqual(actualFile, expectedFile) {
  * given example.
  */
 async function runWithTemplateDir(exampleName, fn) {
-  let suffix = Math.floor(Math.random() * 1000000000000);
-  let newDir = `./test/tmp-projects/${exampleName}-${suffix}`;
+  let newDirPref = `./test/tmp-projects/${exampleName}/tmp-`;
```

Comment:
Can you use the whole word "prefix" instead of "pref"? In my mind, "pref" means "preference".

Also, looks like you added an extra level of nesting here, which is causing the test failure. I'd rather just keep the same nesting level as before (which also make makes it easier to potentially avoid `mkdTempSafe`), but I don't have a strong preference.